### PR TITLE
Updated CONTRIBUTING.md with labels [touch:22]

### PR DESCRIPTION
**Pull request by marcustisater**
@MoOx What do you think? I think it's best not to have too many labels for starters. I combined some of yours, great suggestions btw.

### CODE REVIEW

**Comment by marcustisater** [comment]
@mxstbr feel free to correct any grammar/spelling misstakes or if you would like to add something :wink:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
 <a name="issue-labels"></a>
 ## Issue labels
 
+* `bug` - Report any bugs regarding postcss.com, please be as specific as possible.
+* `discussion` - Open discussion for upcoming feature, question or planning.
```
Comment (mxstbr) [inline CONTRIBUTING.md]: This seems friendlier and inviting:

> Open discussion, please chime in! (E.g. for upcoming features, questions, planning, etc.)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
 <a name="issue-labels"></a>
 ## Issue labels
 
+* `bug` - Report any bugs regarding postcss.com, please be as specific as possible.
+* `discussion` - Open discussion for upcoming feature, question or planning.
+* `feature request` - Request for a feature or increase of improvement.
```
Comment (mxstbr) [inline CONTRIBUTING.md]: > increase of improvement

What does that mean?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
 <a name="issue-labels"></a>
 ## Issue labels
 
+* `bug` - Report any bugs regarding postcss.com, please be as specific as possible.
+* `discussion` - Open discussion for upcoming feature, question or planning.
+* `feature request` - Request for a feature or increase of improvement.
+* `help wanted` - Issues we would need help to resolve.
```
Comment (mxstbr) [inline CONTRIBUTING.md]: Maybe delete the `would`?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
 <a name="issue-labels"></a>
 ## Issue labels
 
+* `bug` - Report any bugs regarding postcss.com, please be as specific as possible.
+* `discussion` - Open discussion for upcoming feature, question or planning.
+* `feature request` - Request for a feature or increase of improvement.
+* `help wanted` - Issues we would need help to resolve.
+* `high-priority` - High prioritized issues that needs to be resolved as soon as possible.
```
Comment (mxstbr) [inline CONTRIBUTING.md]: > High priority issues that need to be resolved as soon as possible.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
 <a name="issue-labels"></a>
 ## Issue labels
 
+* `bug` - Report any bugs regarding postcss.com, please be as specific as possible.
+* `discussion` - Open discussion for upcoming feature, question or planning.
+* `feature request` - Request for a feature or increase of improvement.
+* `help wanted` - Issues we would need help to resolve.
+* `high-priority` - High prioritized issues that needs to be resolved as soon as possible.
+* `low-priority` - Low prioritized issues that are not very important.
```
Comment (mxstbr) [inline CONTRIBUTING.md]: > Low priority issues.

The `that are not important` is already in the `low priority`, no need to repeat that information.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
...
 ## Issue labels
 
+* `bug` - Report any bugs regarding postcss.com, please be as specific as possible.
+* `discussion` - Open discussion for upcoming feature, question or planning.
+* `feature request` - Request for a feature or increase of improvement.
+* `help wanted` - Issues we would need help to resolve.
+* `high-priority` - High prioritized issues that needs to be resolved as soon as possible.
+* `low-priority` - Low prioritized issues that are not very important.
+* `in-progress` - Issues that are currently being worked on and assigned.
+* `needs documentation` - Request for documentation to be needed.
```
Comment (mxstbr) [inline CONTRIBUTING.md]: This one seems self-explanatory enough, do we really need another sentence saying the exact same thing?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
...
+* `discussion` - Open discussion for upcoming feature, question or planning.
+* `feature request` - Request for a feature or increase of improvement.
+* `help wanted` - Issues we would need help to resolve.
+* `high-priority` - High prioritized issues that needs to be resolved as soon as possible.
+* `low-priority` - Low prioritized issues that are not very important.
+* `in-progress` - Issues that are currently being worked on and assigned.
+* `needs documentation` - Request for documentation to be needed.   
+* `needs investigation` - Request for investigation to follow before moving on.
+* `needs revision/testing` - Issues that needs to be reviewed or tested.
+* `PR welcome` - Pull Requests are welcome to be suggested for resolving issue.
```
Comment (mxstbr) [inline CONTRIBUTING.md]: Same as `needs documentation`, does this really need another sentence saying the exact same thing as the label?

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
...
+* `question` - Question regarding postcss.com - not to be used as support for [PostCSS](https://github.com/postcss/postcss).
+* `duplicated` - Duplicated issues that has already been created before.
```
Comment (mxstbr) [inline CONTRIBUTING.md]: > ...that **have**...

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
 <a name="issue-labels"></a>
 ## Issue labels
 
+* `bug` - Report any bugs regarding postcss.com, please be as specific as possible.
+* `discussion` - Open discussion for upcoming feature, question or planning.
+* `feature request` - Request for a feature or increase of improvement.
+* `help wanted` - Issues we would need help to resolve.
+* `high-priority` - High prioritized issues that needs to be resolved as soon as possible.
+* `low-priority` - Low prioritized issues that are not very important.
+* `in-progress` - Issues that are currently being worked on and assigned.
```
Comment (mxstbr) [inline CONTRIBUTING.md]: Isn't it clear that they are assigned if they are being worked on? Redundant information?

**Comment by mxstbr** [comment]
Left a bunch of comments!

Some of these labels are really self-explanatory, it seems like a waste of time to describe them with a sentence saying the exact same thing as the label, right?

**Comment by marcustisater** [comment]
Agreed, thanks.

**Comment by marcustisater** [comment]
All good for first set of labels?

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
 <a name="issue-labels"></a>
 ## Issue labels
 
+* `bug` - Report any bugs regarding postcss.com, please be as specific as possible.
+* `discussion` - Open discussion, please chime in! (E.g. for upcoming features, questions, planning, etc.)
+* `feature request` - Request for a feature or enhancement.
+* `help wanted` - Issues we need help to resolve.
+* `high-priority` - High priority issues that need to be resolved as soon as possible.
+* `low-priority` - Low prioritized issues.
```
Comment (mxstbr) [inline CONTRIBUTING.md]: Change ```low prioritized``` to ```low priority``` like the `high-priority` label

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
 <a name="issue-labels"></a>
 ## Issue labels
 
+* `bug` - Report any bugs regarding postcss.com, please be as specific as possible.
+* `discussion` - Open discussion, please chime in! (E.g. for upcoming features, questions, planning, etc.)
+* `feature request` - Request for a feature or enhancement.
+* `help wanted` - Issues we need help to resolve.
+* `high-priority` - High priority issues that need to be resolved as soon as possible.
```
Comment (mxstbr) [inline CONTRIBUTING.md]: Actually, maybe lets remove the `High priority` and `Low priority` from the descriptions totally and change them to "Issues that need to be resolved asap" and "Issues that don't need urgent solving" or something like that?

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
 <a name="issue-labels"></a>
 ## Issue labels
 
+* `bug` - Report any bugs regarding postcss.com, please be as specific as possible.
+* `discussion` - Open discussion, please chime in! (E.g. for upcoming features, questions, planning, etc.)
+* `feature request` - Request for a feature or enhancement.
+* `help wanted` - Issues we need help to resolve.
+* `high-priority` - High priority issues that need to be resolved as soon as possible.
+* `low-priority` - Low prioritized issues.
+* `in-progress` - Issues that is on-going.
```
Comment (mxstbr) [inline CONTRIBUTING.md]: > Issues that **are ongoing**.

(AFAIK ongoing doesn't have a dash)

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -15,6 +15,21 @@ and [submitting pull requests](#pull-requests).
...
+* `bug` - Report any bugs regarding postcss.com, please be as specific as possible.
+* `discussion` - Open discussion, please chime in! (E.g. for upcoming features, questions, planning, etc.)
+* `feature request` - Request for a feature or enhancement.
+* `help wanted` - Issues we need help to resolve.
+* `high-priority` - High priority issues that need to be resolved as soon as possible.
+* `low-priority` - Low prioritized issues.
+* `in-progress` - Issues that is on-going.
+* `needs documentation`
+* `needs investigation`
+* `needs revision/testing` - Issues that needs to be reviewed or tested.
```
Comment (mxstbr) [inline CONTRIBUTING.md]: > Issues that **need**...

**Comment by MoOx** [comment]
I don't think labels need documentation. Labels are for collaborators only (so you guys) and you probably already understand them.

**Comment by mxstbr** [comment]
^ Good point @MoOx, agreed.

**Comment by marcustisater** [comment]
I agree but it's good to have something to refer with. It's already written, why waste it :smile: 

Thanks though for check-in

**Comment by MoOx** [comment]
You will have to maintain it now...

**Comment by marcustisater** [comment]
Fair point.

**Pull request closed without merging**